core/src/assets/config/config.json

```diff
@@ -1,62 +1,80 @@
 {
   "config": {
-    "features":{
-        "SERVICE_CATALOG":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"servicecatalog.k8s.io"
-              }
-          ]
-        },
-        "SERVICE_CATALOG_ADDONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"servicecatalog.kyma-project.io"
-              }
-          ]
-        },
-        "EVENTING":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"eventing.kyma-project.io"
-              }
-          ]
-        },
-        "API_GATEWAY":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"gateway.kyma-project.io"
-              }
-          ]
-        },
-        "APPLICATIONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"applicationconnector.kyma-project.io"
-              }
-          ]
-        },
-        "ADDONS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"addons.kyma-project.io"
-              }
-          ]
-        },
-        "SERVERLESS":{
-          "selectors":[
-              {
-                "type":"apiGroup",
-                "apiGroup":"serverless.kyma-project.io"
-              }
+    "features": {
+      "SERVICE_CATALOG": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "servicecatalog.k8s.io"
+          }
+        ]
+      },
+      "SERVICE_CATALOG_ADDONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "servicecatalog.kyma-project.io"
+          }
+        ]
+      },
+      "EVENTING": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "eventing.kyma-project.io"
+          }
+        ]
+      },
+      "API_GATEWAY": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "gateway.kyma-project.io"
+          }
+        ]
+      },
+      "APPLICATIONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "applicationconnector.kyma-project.io"
+          }
+        ]
+      },
+      "ADDONS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "addons.kyma-project.io"
+          }
+        ]
+      },
+      "SERVERLESS": {
+        "selectors": [
+          {
+            "type": "apiGroup",
+            "apiGroup": "serverless.kyma-project.io"
+          }
+        ]
+      },
+      "OBSERVABILITY": {
```

Comment:
we should add the `isEnabled` flag here so if someone doesn't want the default links, he/she can disable it

Reply:
the second question would be if we should even have the defaults? we want to create the product for all Kubernetes clusters, not only Kyma, so maybe we shouldn't have anything if those are not specifically defined?

Reply:
Also, please add those to our `configmap.yaml`

Reply:
@valentinvieriu

Reply:
yes, we should add `isEnabled` on all our features

Reply:
I agree to @dariadomagala . Our defaults should work with any K8s cluster by default. We should use the overwrites to enable the features that are specific to Kyma.

Reply:
Sooo... I should remove it from `config.json` and leave it only in `configmap.yaml`? If yes, I'm afraid our developer experience will suffer as we'll have to manually add this section to our `config.json` and never include it in a commit..

Reply:
Maybe we could keep it in the config map just with the `isEnabled` flag set to false by default and change it in the config map if needed?